src/apis/TraktSearch.ts

```diff
@@ -220,27 +220,28 @@ class _TraktSearch extends TraktApi {
 				status: 404,
 				text: responseText,
 				extra: {
-					item: Item.save(item),
+					item: item.save(),
 				},
 			});
 		}
 		return searchItem;
 	}
 
 	async findShow(
-		itemOrUrl: Item | string,
+		itemOrUrl: EpisodeItem | string,
 		caches: CacheItems<['traktItems', 'urlsToTraktItems']>,
 		cancelKey = 'default'
 	): Promise<TraktSearchShowItem> {
-		const showUrl =
-			itemOrUrl instanceof Item
-				? `${itemOrUrl.type}?query=${encodeURIComponent(itemOrUrl.title)}`
-				: itemOrUrl;
+		const showUrl = isItem(itemOrUrl)
+			? `show?query=${encodeURIComponent(itemOrUrl.show.title)}`
+			: itemOrUrl;
 		let traktDatabaseId = caches.urlsToTraktItems.get(showUrl);
-		let cacheItem = traktDatabaseId ? caches.traktItems.get(traktDatabaseId) : null;
+		let cacheItem = traktDatabaseId
+			? (caches.traktItems.get(traktDatabaseId) as TraktShowItemValues | undefined)
+			: null;
 		if (!cacheItem) {
 			let show;
-			if (itemOrUrl instanceof Item) {
+			if (isItem(itemOrUrl)) {
 				show = ((await this.findItem(itemOrUrl, cancelKey)) as TraktSearchShowItem).show;
```

Comment:
It seems to never search for the show. In this location it actually searches for the episode, but it should search for the show. (I think it would be itemOrUrl.show)

Reply:
Fixed